packages/libraries/cli/src/base-command.ts

```diff
@@ -304,6 +330,29 @@ class ClientError extends Error {
   }
 }
 
+export type Flags<T extends typeof Command> = Interfaces.InferredFlags<
```

Comment:
In general I try to bias toward a "literate" style of code flow when possible meaning higher level concepts precede lower ones. In practice that looks like references progressing downward which is what I've done here.